### PR TITLE
feat.create,archive,delete

### PR DESCRIPTION
**Pull request by Shrinkhlarajpoot**
Added features 
-Add to Archive
-Remove from Archive
-Add to Trash
-Remove from Trash

### CODE REVIEW

**Review by KaustubhM1997 — APPROVED** [review]
The UI looks neat. Good to merge!

File: src/frontend/context/notesContext.jsx [hunk 1]
```diff
@@ -0,0 +1,269 @@
...
+
+const NotesProvider = ({ children }) => {
+  const {
+    auth: { token },
+  } = useAuth();
+  const [notesState, notesDispatch] = useReducer(
+    NotesReducerFun,
+    InitialValues
+  );
+  // console.log(notesState.trashList)
```
Comment (Chakravarthi0) [inline src/frontend/context/notesContext.jsx:28]: You can remove this comment

**Review by Chakravarthi0 — APPROVED** [review]
suggestion: TrashCard.css is an empty file, you can remove it, if it won't be needed
rest looks good

File: src/frontend/context/notesContext.jsx (hunk 1 above)
Comment (GeetaPogul) [reply src/frontend/context/notesContext.jsx:28]: ya

File: src/frontend/context/notesContext.jsx [hunk 2]
```diff
@@ -0,0 +1,269 @@
...
+            token,
+            note
+          );
+          console.log(data);
+
+          if (status === 201) {
+            notesDispatch({
+              type: "SET_TRASH_NOTES",
+              payload: {
+                // notesList: data.notes,
```
Comment (GeetaPogul) [inline src/frontend/context/notesContext.jsx:135]: here also

Comment (GeetaPogul) [reply src/frontend/context/notesContext.jsx:135]: if you don't need this

File: src/index.js [hunk 3]
```diff
@@ -3,7 +3,7 @@ import ReactDOM from "react-dom";
 import "./index.css";
 import App from "./App";
 import { makeServer } from "./server";
-import { AuthProvider } from "./frontend/context";
+import { AuthProvider, NotesProvider } from "./frontend/context";
 import { BrowserRouter } from "react-router-dom";
 
 // Call make Server
```
Comment (GeetaPogul) [inline src/index.js:9]: you can remove unwanted comment.

**Review by GeetaPogul — COMMENTED** [review]
good to be merge

**Pull request merged**